components/temi/sources/common.mjs

```diff
@@ -0,0 +1,75 @@
+import app from "../temi.app.mjs";
+import constants from "../common/constants.mjs";
+import utils from "../common/utils.mjs";
+
+export default {
+  props: {
+    app,
+    http: "$.interface.http",
```

Comment:
I think it's preferred that we put "common" files in their own directory. I would move `sources/common.mjs` to `sources/common/common.mjs` and update the import statements here and in `new-transcription`.